test/fuzz-reader/reader_test.go

```diff
@@ -8,29 +8,34 @@ import (
 	"io/ioutil"
 	"os"
 	"path/filepath"
+	"runtime"
 	"testing"
 )
 
 func TestCorpusSymlinks(t *testing.T) {
-	fds, err := ioutil.ReadDir("corpus")
-	if err != nil {
-		t.Fatal(err)
-	}
-	if len(fds) == 0 {
-		t.Fatal("no file descriptors found in corpus/")
-	}
 
-	for i := range fds {
-		if fds[i].Mode()&os.ModeSymlink != 0 {
-			if path, err := os.Readlink(filepath.Join("corpus", fds[i].Name())); err != nil {
-				t.Errorf("broken symlink: %v", err)
-			} else {
-				if _, err := os.Stat(filepath.Join("corpus", path)); err != nil {
+	// avoid symbolic link error on windows
+	if runtime.GOOS != "windows" {
```

Comment:
Could we just skip the test early?

```go
if runtime.GOOS == "windows" {
	t.Skip("git symlinks aren't supported on windows")
} 
```

Also, I was going to put this in a different PR as the other change isn't related to this test failure.

Reply:
I'm ok if you want to change where the skip is under another PR.